src/solve.jl

```diff
@@ -75,7 +75,7 @@ function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algTyp
                                                        ode_prob,
                                                        OrdinaryDiffEq.alg_order(alg)))
     end
-    integrator.dt = dt
+    integrator.dt = zero(dt)
```

Comment:
> Not this line though, just the PR in general?

Yes just the PR in general.

> Nah, it makes sense. Just was curious about the practical effect. In theory it would do a constant extrapolation if dt=0, I just was wondering if that ever came into play.

But it is not implemented yet, is it? As far as I see, all calls of `current_interpolant` and `current_interpolant!` (and also of `current_extrapolant` and `current_extrapolant!`) will calculate a `Θ` that is not a number for `dt=0` - which will always lead to interpolations/extrapolations that are not a number, I guess.